src/OV/OpenVarioMenu.cpp

```diff
@@ -353,7 +1064,7 @@ MainMenuWidget::Prepare([[maybe_unused]] ContainerWindow &parent,
 
     TWidgetDialog<FileMenuWidget>
       sub_dialog(WidgetDialog::Full{}, dialog.GetMainWindow(),
-                 GetLook(), "OpenVario File");
+                 GetLook(), "OpenVario Files");
```

Comment:
Don't hide unrelated changes in this commit - if you wish to change labels, that should be a new commit

Reply:
OK